src/react/HomePage.jsx

```diff
@@ -84,7 +86,11 @@ export default ({ astarte, history }) => {
         <WaitForData data={devicesStats.value} status={devicesStats.status}>
           {({ connected_devices: connectedDevices, total_devices: totalDevices }) => (
             <Col xs={6} className={cellSpacingClass}>
-              <DevicesCard connectedDevices={connectedDevices} totalDevices={totalDevices} />
+              <DevicesCard
+                connectedDevices={connectedDevices}
+                totalDevices={totalDevices}
+                astarte={astarte}
```

Comment:
i'm not to fond of passing the astarte client down the view. The DeviceCard only needs it for chart purposes, so it's better to narrow the scope of the parameter.
In this instance you can just forward a `connectedDevicesProvider`

Reply:
Right, fixed it